cypress/e2e/with_api/catalogueCategory.cy.ts

```diff
@@ -0,0 +1,323 @@
+    ).should('have.length', 2);
+    cy.go('back');
+  }
+};
+
+const deleteCatalogueCategory = (name: string) => {
+  cy.intercept({
+    method: 'DELETE',
+    url: '**/catalogue-categories/*',
+  }).as('getCatalogueCategoryData');
+  cy.findByRole('button', {
+    name: `actions ${name} catalogue category button`,
+  }).click();
+
+  cy.findByRole('menuitem', {
+    name: `delete ${name} catalogue category button`,
+  }).click();
+
+  cy.findByRole('button', { name: 'Continue' }).click();
+  cy.wait('@getCatalogueCategoryData', { timeout: 10000 });
+};
+
+const saveAsCatalogueCategory = (name: string) => {
+  cy.intercept({
+    method: 'POST',
+    url: '**/catalogue-categories',
+  }).as('getCatalogueCategoryData');
+  cy.findByRole('button', {
+    name: `actions ${name} catalogue category button`,
+  }).click();
+
+  cy.findByRole('menuitem', {
+    name: `save as ${name} catalogue category button`,
+  }).click();
+
+  cy.findByRole('button', { name: 'Save' }).click();
+  cy.wait('@getCatalogueCategoryData', { timeout: 10000 });
+  cy.findByText(`${name}_copy_1`).should('exist');
+};
+
+const copyToCatalogueCategory = (values: { checkedCategories: string[] }) => {
+  cy.intercept({
+    method: 'POST',
+    url: '**/catalogue-categories',
+  }).as('getCatalogueCategoryData');
+
+  for (let i = 0; i < values.checkedCategories.length; i++) {
+    cy.findByLabelText(`${values.checkedCategories[i]} checkbox`).click();
+  }
+  cy.findByRole('button', { name: 'Copy to' }).click();
+  cy.findByRole('button', { name: 'navigate to catalogue home' }).click();
+  cy.findByRole('button', { name: 'Copy here' }).click();
+  cy.wait('@getCatalogueCategoryData', { timeout: 10000 });
+  cy.findByRole('button', { name: 'navigate to catalogue home' }).click();
+  for (let i = 0; i < values.checkedCategories.length; i++) {
+    cy.findByText(`${values.checkedCategories[i]}`).should('exist');
+    deleteCatalogueCategory(`${values.checkedCategories[i]}`);
+  }
+};
+
+const moveToCatalogueCategory = (values: { checkedCategories: string[] }) => {
+  cy.intercept({
+    method: 'PATCH',
+    url: '**/catalogue-categories/*',
+  }).as('getCatalogueCategoryData');
+
+  for (let i = 0; i < values.checkedCategories.length; i++) {
+    cy.findByLabelText(`${values.checkedCategories[i]} checkbox`).click();
+  }
+  cy.findByRole('button', { name: 'Move to' }).click();
+  cy.findByRole('button', { name: 'navigate to catalogue home' }).click();
+  cy.findByRole('button', { name: 'Move here' }).click();
+  cy.wait('@getCatalogueCategoryData', { timeout: 10000 });
+  cy.findByRole('button', { name: 'navigate to catalogue home' }).click();
+  for (let i = 0; i < values.checkedCategories.length; i++) {
+    cy.findByText(`${values.checkedCategories[i]}`).should('exist');
+  }
+};
+
+const addCatalogueCategories = () => {
+  modifyCatalogueCategory({
+    name: 'Lensess',
```

Comment:
Should this not be `Lenses` and then  the edit rename it to something like `Lenses 2`, since presumably this will also be used in catalogue items/items tests without the edit correcting the name.